arboretum/permissions/checks/test_org_permissions.py

```diff
@@ -0,0 +1,109 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2021 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Repository organization permissions check."""
+
+from compliance.check import ComplianceCheck
+from compliance.evidence import DAY, ReportEvidence, evidences
+from compliance.utils.data_parse import get_sha256_hash
+
+
+class OrgPermissionsCheck(ComplianceCheck):
+    """Monitor repository organization permissions."""
+
+    @property
+    def title(self):
+        """
+        Return the title of the checks.
+
+        :returns: the title of the checks
+        """
+        return 'Report on Repository Organization Permissions'
```

Comment:
To be consistent with the other check...

```suggestion
        return 'Repository Organization/Owner Permissions'
```

Reply:
Done